test/integration_test.go

```diff
@@ -1453,6 +1453,7 @@ func (ts *IntegrationTestSuite) TestInterceptorCalls() {
 		"WorkflowOutboundInterceptor.RequestCancelExternalWorkflow": {},
 		"WorkflowOutboundInterceptor.SignalExternalWorkflow":        {},
 		"WorkflowOutboundInterceptor.UpsertSearchAttributes":        {},
+		"WorkflowOutboundInterceptor.UpsertMemo":                    {},
```

Comment:
Can you provide a full integration test here? I think a good test would be to start a workflow with some memo, assert that memo, then upsert that memo (and set a query saying you've done so), then describe the workflow via the client and confirm the memo was upserted.

If this is not available in the server yet for an integration test, I don't think we should merge the SDK functionality until it is.